### DISCUSSION TITLE
Cinema mod fails on a map where Autoplay also fails

### DISCUSSION THREAD

**Discussion by JacksonChen666** [ppy/osu#26032] [Q&A] [answered]
An [Aspire map which causes Autoplay to fail](https://osu.ppy.sh/beatmapsets/594751#osu/1257904) will also cause the Cinema mod to fail, which isn't a good experience as it disrupts watching the video/storyboard.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/26080
